engine/fscdr.go

```diff
@@ -86,9 +87,12 @@ func (fsCdr FSCdr) getExtraFields() map[string]string {
 			origFieldVal = fsCdr.searchExtraField(field.Id, fsCdr.body)
 		}
 		if parsed, err := field.Parse(origFieldVal); err == nil {
+			if alias, has := extraFieldsAliases[field.Id]; has && alias != utils.EmptyString {
+				extraFields[alias] = parsed
```

Comment:
Please unite both into one, no need of separate continue inside since it is a bit confusing. I would go like:
fldName = field.ID
if hasAlias -> {fldName = alias}
extraFields[fldName] = parsed